### PR TITLE
Fix timeout for post-merge aws functional tests.

### PR DESCRIPTION
**Pull request by gurevichdmitry**
### Summary of your changes

- Cloudbeat period was increased to 600 seconds.
- CI action (cloudbeat deployment) waiting timeout was increased to 600s.
- Fixed just commands to support `kind-mono` and `kind-multi` deployments

### CODE REVIEW

File: .github/workflows/k8s-ci.yml [hunk 1]
```diff
@@ -328,9 +328,19 @@ jobs:
...
 
+      - name: Set Kind Configuration
+        if: matrix.test-target == 'aws'
+        run: |
+          echo "KIND_CONFIG=kind-mono" >> $GITHUB_ENV
+
+      - name: Set Kind Configuration
+        if: matrix.test-target != 'aws'
+        run: |
+          echo "KIND_CONFIG=kind-multi" >> $GITHUB_ENV
```
Comment (olegsu) [inline .github/workflows/k8s-ci.yml:339]: Can we do it in one step?

**Review by olegsu — CHANGES_REQUESTED** [review]

**Comment by olegsu** [comment]
Do we have a plan for how to overcome this? waiting for 10 min seems like something we want to avoid.

File: .github/workflows/k8s-ci.yml (hunk 1 above)
Comment (gurevichdmitry) [reply .github/workflows/k8s-ci.yml:339]: @olegsu github actions do not support "if else", I can implement inline script for this functionality.

**Comment by gurevichdmitry** [comment]
> Do we have a plan for how to overcome this? waiting for 10 min seems like something we want to avoid.

@olegsu, Not for this case. Fetching resources from AWS depends on some factors - network, cloud services availability, resources count, etc. In current context, the fetching max time is limited to 10 minutes. By design, timeouts are provided at all stages, and customization depends on what we are trying to reach. Currently the requirement is to execute aws functional tests on post-merge and we need to retrieve stable results (not flaky). AWS functional tests run takes around 17 minutes and executed in parallel with other test jobs. I think this is acceptable time and improvement is not required in this case.

**Review by olegsu — APPROVED** [review]

**Pull request merged**